posts/2022-01-31-configuring-ocp-for-user-namespace.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Configuring ocp for user namespace
+published_date: "2022-01-31 11:45:58 +0000"
+layout: default.liquid
+is_draft: false
+---
+Preparing an openshift cluster for using user namespaces involve
+several steps and hands-over. To simplify the process we are using
+some configurations at freeipa-kustomize that make easier that task.
+
+**Pre-requisites**:
+
+- A 4.9+ OpenShift cluster.
```

Comment:
The procedure works for 4.9 but I'm not sure about 4.10, which uses different mechanisms to enable and activate user namespaces support in CRI-O.

Reply:
Thanks! I am going to check with a 4.10 cluster.

Reply:
I have tried in 4.10 with no experimental rpms installed and it worked, allowing to use the user namespaces into the pod; it mapped in 4.9 and 4.10 the range `0 200000 65536`.The range mapped could differ from one pod to other.